CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+* the SPOs approve of the action (the stake controlled by the SPOs who vote 'yes' meets a certain threshold over those who vote 'no').
+
+> **Warning**
+> As explained below, different stake distributions apply to DReps and SPOs.
+
+##### Requirements
+
+The following table details the ratification requirements for each governance action scenario. The columns represent:
+
+* **Governance Action Type**<br/>
+  The type of governance action. Note that the three possible treasury actions involve Lovelace amounts $T_0$, $T_1$, $T_2$, and $T_3$.
+
+* **Constitutional Committee**<br/>
+  A value of :heavy_check_mark: indicates that `Quorum-Many` Constitutional Committee 'yes' votes are required.<br/>
+  A value of :x: means that Constitutional Committee votes do not apply.
+
+* **DReps**<br/>
+  The DRep vote threshold that must be met as a percentage of *active voting stake*, ranging from 0 to 100 (inclusive).
+
+* **AVST**<br/>
+  The _**A**ctive **V**oting **S**take **T**hreshold_. The percentage to be used to determine if there is *sufficient active voting stake*. SPO vote endorsement is not required for the specific governance action if there is a sufficient active voting stake.
+  Otherwise, SPO vote endorsement is required. The :x: symbol indicates that the AVST will be ignored, and only the DRep and SPO votes will be considered regardless of the AVST.
+
+* **SPOs**<br/>
+  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools. The SPO vote is only considered if the AVST threshold is :x: or the AVST is below the AVST threshold.
+
+| Governance Action Type                               | Constitutional Committee | DReps    | AVST     | SPOs     |
+| ---                                                  | :---:                    | ---      | ---      | ---      |
+| 1. Motion of no-confidence                           | :x:                      | $P_1$    | :x:      | $R_1$    |
+| 2(a). New Committee/quorum (_normal state_)          | :heavy_check_mark:       | $P_{2a}$ | $Q_{2a}$ | $R_{2a}$ |
+| 2(b). New Committee/quorum (_state of no-confidence_)| :x:                      | $P_{2b}$ | :x:      | $R_{2b}$ |
+| 3. Update to the Constitution                        | :heavy_check_mark:       | $P_3$    | $Q_3$    | $R_3$    |
+| 4. Hard-Fork initiation                              | :heavy_check_mark:       | $P_4$    | :x:      | $R_4$    |
+| 5. Protocol parameter changes                        | :heavy_check_mark:       | $P_5$    | $Q_5$    | $R_5$    |
+| 6(a). Treasury withdrawal, $[T_0, T_1)$              | :heavy_check_mark:       | $P_{6a}$ | $Q_{6a}$ | $R_{6a}$ |
+| 6(b). Treasury withdrawal, $[T_1, T_2)$              | :heavy_check_mark:       | $P_{6b}$ | $Q_{6b}$ | $R_{6b}$ |
+| 6(c). Treasury withdrawal, $[T_2, T_3)$              | :heavy_check_mark:       | $P_{6c}$ | $Q_{6c}$ | $R_{6c}$ |
```

Comment:
I'm not sure how useful these intervals are. Let's say I want to withdraw `3X` Ada, where `X < T_1` but `2X > T_1`. If I'm confident I can clear `P_{6a}`, but not so much about `P_{6b}`, I'd simply make up some valid sounding reason why I need my payment in 3 instalments.

Also, are `T_0` and `T_3` supposed to be actual parameters? I think it would make sense to just have them be 0 and infinity (or max ada).

Reply:
Given that at most one action of each type can be ratified per epoch, splitting it up would at least have the effect of slowing down the total withdrawal. As the stake distribution changes, it is also not guaranteed that the vote would be the same each epoch. I do like @michael-liesenfelt 's suggestion (which I've added to the CIP below) regarding basing the thresholds on the amount (ie a more continuous model).

Reply:
Is there a reason why there shouldn't be more than one treasury withdrawal possible in the same epoch?

Reply:
of all the actions, the treasury withdrawals have the least strong argument for limiting them (the case for protocol parameter updates, for example, is very strong, since they interact with each other). the reasons are:

* uniformity with the other actions
* forcing the withdrawals to be bundled so that it is clear how much lovelace is leaving the treasury in a given epoch and so that the DReps are not too bombarded with these actions

I guess the downside is that a treasury withdrawal action could fail to ratify because of one problematic account, but effecting everyone for that epoch.

Reply:
> I do like @michael-liesenfelt 's suggestion (which I've added to the CIP below) regarding basing the thresholds on the amount (ie a more continuous model).

@JaredCorduan , I checked the document and I couldn't find mention of Tau-prime, the maximum amount of accumulated ADA available for withdrawal. It seems that the reserve payout rate Rho is currently 0.003. I was mistaken earlier when I suggested a value for Tau'=0.010.

I recommend setting Tau' such that the withdrawal limit increase roughly matches the additions to the treasury per epoch. This would begin the system with an equilibrium Tau' where treasury additions roughly equal to the withdrawal limit change change per epoch.

Additions per epoch = 0.2 * 2.5e7 = 5e6
Withdrawal limit increase per epoch = Tau' * Treasury = **0.005** * 1e9 = 5e6

A starting value of Tau' = **0.005** would ensure that each epoch an additional ~5e6 ADA would be available for withdrawal from the Treasury. Naturally, if a withdrawal is made then the withdrawal limit is proportionally decreased.

Reply:
I have not yet added anything quite as prescriptive as what you are suggesting, but made a note of the topic in general:

https://input-output-rnd.slack.com/archives/GJGPC1QA3/p1669973606029659?thread_ts=1669973559.536239&cid=GJGPC1QA3

There are a few interesting things to still think about with respect to this action, and they might have interactions.